applications/StructuralMechanicsApplication/tests/pendulus_test/pendulus_UL_test.mdpa

```diff
@@ -0,0 +1,307 @@
+Begin ModelPartData
```

Comment:
Make it smaller. Also ideally create the test without mode, to the style of the patch test

Reply:
mdpa?

Reply:
Yes mdpa

Reply:
I prefer this way, but if this make you happy...